public/js/icinga/behavior/modal.js

```diff
@@ -109,6 +109,11 @@
         req.addToHistory = false;
         req.$redirectTarget = $modal.data('redirectTarget');
         req.done(function (data, textStatus, req) {
+            var title = req.getResponseHeader('X-Icinga-Title');
+            if (typeof title !== undefined) {
```

Comment:
The `typeof` instruction returns a string. `typeof title !== 'undefined'` would work. But this won't just be `undefined` if the header is missing. The header might exist, but it might not have a value. Use `!! title` instead. This way you'll continue only if whatever it is can be casted to a boolean true.